.expeditor/scripts/shared.sh

```diff
@@ -272,3 +272,22 @@ fastly_token() {
           -field=token \
           account/static/fastly/eng-services-ops
 }
+
+latest_release_tag() {
+  local repo="${1?repo argument required}"
+  tag=$(curl --silent "https://api.github.com/repos/${repo}/releases/latest" | jq -r .tag_name)
+  echo "${tag}"
+}
+
+install_hub() {
+  local tag
+  tag=$(latest_release_tag github/hub)
+  tag_sans_v="${tag//v/}"
+  url="https://github.com/github/hub/releases/download/${tag}/hub-linux-amd64-${tag_sans_v}.tgz"
```

Comment:
Wow, I didn't realized they have move hub from a ruby gem to a go binary. That's pretty awesome. I wonder if it would be worth packaging that up so we can simplify this to  `hab pkg install` it? (not for this pr, just a thought)

Reply:
oooh, that's a neat idea.

Reply:
Thanks! I added a TODO here for creating a Hab core plans pkg in https://github.com/habitat-sh/habitat/pull/7258/commits/bffee7f207007f63f7e71ed133dbd4acff3647ee